packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -558,6 +559,53 @@ describe('SelectedNetworkController', () => {
         });
       });
     });
+    describe('when the domain is a snap (starts with "npm:" or "local:")', () => {
+      it('returns a proxied globally selected networkClient and does not create a new proxy in the domainProxyMap', () => {
+        const { controller, domainProxyMap, messenger } = setup({
+          state: {
+            domains: {},
+          },
+          useRequestQueuePreference: true,
+        });
+        jest.spyOn(messenger, 'call');
+        const snapDomain = 'npm:@metamask/bip32-example-snap';
+        const result = controller.getProviderAndBlockTracker(snapDomain);
```

Comment:
Nit: Typically we try to separate the "execution" phase of the test from the setup and assertions, so that it's clearer what the test is meant to test. See here for more on that: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#-highlight-the-exercise-phase

I think in this case, it would mean adding newlines around this line:
```suggestion

        const result = controller.getProviderAndBlockTracker(snapDomain);

```

And I think we could make the same change to the other two tests as well